x-pack/plugins/security_solution/public/common/components/event_details/insights/related_alerts_by_session.tsx

```diff
@@ -43,6 +43,7 @@ export const RelatedAlertsBySession = React.memo<Props>(
       timelineId: timelineId ?? '',
       signalIndexName: null,
       includeAlertIds: true,
+      ignoreTimerange: true,
```

Comment:
We might want to do the same for related alerts by source event ✅

Reply:
Ah, does ancestry take the time range into account atm?

Reply:
ya removed in https://github.com/elastic/kibana/pull/141137/commits/e0fd327d2f45b62d9142a7d0d8a5ea8aa8ca4a42